Documentation/ceph-cluster-crd.md

```diff
@@ -1033,7 +1033,7 @@ spec:
                   operator: In
                   values:
                     - cluster1
-                topologyKey: "topology.kubernetes.io/zone"
+              topologyKey: "topology.kubernetes.io/zone"
```

Comment:
a couple of other places where this can be fixed - [1](https://github.com/rook/rook/blob/master/design/ceph/storage-class-device-set.md#example-1-aws-cross-az), [2](https://github.com/rook/rook/blob/master/design/ceph/storage-class-device-set.md#example-3-different-resource-needs) 

I would suggest fixing them in the same PR.